app/src/main/java/com/hedvig/app/feature/marketing/pickmarket/PickMarketScreen.kt

```diff
@@ -47,13 +49,29 @@ import com.hedvig.app.feature.settings.Market
 import com.hedvig.app.ui.compose.composables.buttons.LargeContainedButton
 import com.hedvig.app.ui.compose.theme.HedvigTheme
 import com.hedvig.app.ui.compose.theme.separator
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.isActive
 import kotlinx.coroutines.launch
+import kotlin.time.Duration.Companion.milliseconds
 
 enum class PickMarketSheet {
     MARKET,
     COUNTRY,
 }
 
+@Suppress("NOTHING_TO_INLINE")
+@Composable
+inline fun ForceRecompositions() {
+    val bool by produceState(false) {
+        while (isActive) {
+            value = !value
+            delay(1000.milliseconds)
+        }
+    }
+    val f = animateFloatAsState(if (bool) 0f else 1f)
+    println(f.value)
+}
+
```

Comment:
Oh we did kept this, probably by accident right?

Personally, I'd love to have this handy in the codebase, if placed in an appropriate package and not used from production code, because when doing UI work I might want to use it real quick. Just like with another function that I pretty much always add while working, and delete before commit which is:
```kotlin
fun Modifier.debugBorder(color: Color = Color.Red, width: Dp = 1.dp): Modifier {
    return this.then(Modifier.border(width, color))
}
```
So as I said in the office as well, I'd not oppose having a place for such non-production functions, but otherwise just delete this :D

Reply:
it was accidental, yeah 😬